cedar-policy-generators/src/policy.rs

```diff
@@ -30,6 +30,14 @@ pub struct GeneratedPolicy {
     abac_constraints: Expr,
 }
 
+impl From<GeneratedPolicy> for est::Policy {
+    fn from(gp: GeneratedPolicy) -> est::Policy {
+        let sp: StaticPolicy = gp.into();
+        let p: Policy = sp.into();
+        p.into()
```

Comment:
this is simultaneously beautiful and gross

Reply:
I mean 35 and 36 compile into no-ops, but I do wish the type system could figure out transitive into's